db/migrate/20250821130409_add_attendance_days_to_grade_entries.rb

```diff
@@ -0,0 +1,5 @@
+class AddAttendanceDaysToGradeEntries < ActiveRecord::Migration[8.0]
+  def change
+    add_column :grade_entries, :attendance_days, :bigint
```

Comment:
Per the client's request, we'll change the model to support the attendance days values.